src/System.IO.Compression/src/System/IO/Compression/Crc32Helper.cs

```diff
@@ -495,5 +529,11 @@ static public uint UpdateCrc32(uint crc32, byte[] buffer, int offset, int length
             crc32 ^= 0xFFFFFFFFU;
             return crc32;
         }
+
+        [SecurityCritical]
```

Comment:
Why the [SecurityCritical] attribute?  These attributes aren't actually verified in corefx, and if they were, this would likely fail due to being called from a method that's not SecurityCritical or SecuritySafeCritical.